asb-ballerina/messageReceiver.bal

```diff
@@ -145,35 +147,41 @@ public isolated client class MessageReceiver {
     # + sequenceNumber - Unique number assigned to a message by Service Bus. The sequence number is a unique 64-bit
     #                    integer assigned to a message as it is accepted and stored by the broker and functions as
     #                    its true identifier.
-    # + return - An `asb:Error` if failed to receive deferred message, a Message record if successful or else `()`
+    # + return - An error if failed to receive deferred message, a Message record if successful or else `()`
     @display {label: "Receive Deferred Message"}
     isolated remote function receiveDeferred(@display {label: "Deferred Msg Seq Num"} 
                                              int sequenceNumber) 
-                                             returns @display {label: "Deferred Message"}  Message|Error? {
+                                             returns @display {label: "Deferred Message"}  Message|error? {
         Message? message = check receiveDeferred(self.receiverHandle, self, sequenceNumber);
         return message;
     }
 
     # The operation renews lock on a message in a queue or subscription based on messageLockToken.
     # 
     # + message - `asb:Message` record
-    # + return - An `asb:Error` if failed to renew message or else `()`
+    # + return - An error if failed to renew message or else `()`
     @display {label: "Renew Lock On Message"}
-    isolated remote function renewLock(@display {label: "Message"} Message message) returns Error? {
+    isolated remote function renewLock(@display {label: "Message"} Message message) returns error? {
         if message?.lockToken.toString() != DEFAULT_MESSAGE_LOCK_TOKEN {
             return renewLock(self.receiverHandle, self, message?.lockToken.toString());
         }
-        return error Error("Failed to renew lock on message with ID " + message?.messageId.toString());
+        return error("Failed to renew lock on message with ID " + message?.messageId.toString());
     }
 
     # Closes the ASB sender connection.
     #
-    # + return - An `asb:Error` if failed to close connection or else `()`
+    # + return - An error if failed to close connection or else `()`
     @display {label: "Close Receiver Connection"}
-    isolated remote function close() returns Error? {
+    isolated remote function close() returns error? {
         return closeReceiver(self.receiverHandle);
     }
 
+    isolated remote function getApplicationPropertyByName(Message message, string name) returns any|error? {
```

Comment:
This is not actually a remote function. Shall we consult Ballerina team on this?

Reply:
As this is a util method, shall we move it from the Client?